src/views/product-landing/components/heading/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { HeadingProps } from './types'
+import clamp from 'lib/clamp'
+import s from './style.module.css'
+
+function Heading({
+  slug,
+  heading,
+  level,
+  size,
+}: HeadingProps): React.ReactElement {
+  const Component = `h${level}` as React.ElementType
+  return (
+    <Component
+      id={slug}
+      data-style-as={size}
```

Comment:
What's this property used for?

Reply:
Nothing, thanks for catching this - was cruft from earlier debugging 😅 Removed! ✅

Reply:
TIL that cruft is a word!